Description: Avoi updating copyright year during build
 This fixes reproducible debci
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-10-10

--- a/build/banner.js
+++ b/build/banner.js
@@ -6,7 +6,7 @@
 function getBanner(pluginFilename) {
   return `/*!
   * Bootstrap${pluginFilename ? ` ${pluginFilename}` : ''} v${pkg.version} (${pkg.homepage})
-  * Copyright 2011-${year} ${pkg.author}
+  * Copyright 2011-2020 ${pkg.author}
   * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
   */`
 }
